.yamato/config/__shared.metafile

```diff
@@ -252,26 +353,66 @@ platforms:
         type: Unity::VM
         image: mobile/android-execution-r19:stable
         flavor: b1.large
+    commands:
+      [OpenGLES3, Vulkan]:
+        setup_utr: 
+          - curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr.bat --output utr.bat
+        call_utr:
+          - |5-
+                    set ANDROID_DEVICE_CONNECTION=%BOKKEN_DEVICE_IP%
+                    git rev-parse HEAD | git show -s --format=%%cI > revdate.tmp
+                    set /p GIT_REVISIONDATE=<revdate.tmp
+                    echo %GIT_REVISIONDATE%
+                    del revdate.tmp
+                    utr <UTR_FLAGS>
+        call_utr_without_git:
+          - utr <UTR_FLAGS>
+        unity_downloader_cli:
+          - pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
+          - unity-downloader-cli <EDITOR_REVISION_CMD> -p WindowsEditor -c editor -c il2cpp -c android  --wait --published-only
+        turn_off_firewall: 
+          - NetSh Advfirewall set allprofiles state off
+        connect_bokken:
+          - '%ANDROID_SDK_ROOT%\platform-tools\adb.exe connect %BOKKEN_DEVICE_IP%'
+          - powershell %ANDROID_SDK_ROOT%\platform-tools\adb.exe devices
+        kill_server: 
+          - start %ANDROID_SDK_ROOT%\platform-tools\adb.exe kill-server
+
   iPhone:
     name: iPhone
-    os: ios
+    os: macos
     components:
       - editor
       - iOS
       - il2cpp
     agents_project:
       default:
         type: Unity::mobile::iPhone
-        image: mobile/macos-10.15-testing:stable
+        image: mobile/macos-10.13-testing:stable
         flavor: b1.medium
       standalone_build:
         type: Unity::VM::osx
-        image: mobile/macos-10.15-testing:stable
+        image: mobile/macos-10.13-testing:stable
         flavor: b1.large
+    commands:
+      [Metal]:
+        setup_utr:
+          - curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr --output utr
+          - chmod +x ./utr
+        call_utr:
+          - |5-
+                    export GIT_REVISIONDATE=`git rev-parse HEAD | git show -s --format=%cI`
+                    ./utr <UTR_FLAGS>
+        call_utr_without_git:
+          - ./utr <UTR_FLAGS>
+        unity_downloader_cli:
+          - brew tap --force-auto-update unity/unity git@github.cds.internal.unity3d.com:unity/homebrew-unity.git
+          - brew install unity/unity/unity-downloader-cli
+          - unity-downloader-cli <EDITOR_REVISION_CMD> -c editor -c iOS -c il2cpp  --wait --published-only
 non_project_agents:
   cds_ops_ubuntu_small:
     type: Unity::VM
-    image: package-ci/ubuntu:stable
+    image: cds-ops/ubuntu-16.04-base:stable
```

Comment:
Merge typo I think, but we need to keep the package-ci one

Reply:
I've kept this one so it matches its 'title' or 'key', but I made all jobs use `package_ci_ubuntu_small` which uses package-ci, but I'll doublecheck

Reply:
oh, I see. Then maybe we can just remove this `cds_ops_ubuntu_small` entry? If it's not use by any job

Reply:
good idea, removed it